src/groovy/au/org/emii/gogoduck/job/JobPresenter.groovy

```diff
@@ -0,0 +1,22 @@
+package au.org.emii.gogoduck.job
+
+/**
+ * Adds in a few extra useful attributes for a client.
+ */
+class JobPresenter extends HashMap {
+    JobPresenter(job, jobExecutorService, jobStoreService, createLink) {
+        super(job.properties)
+
+        if (job.status == Status.NEW) {
```

Comment:
I'd personally prefer this function to be a `switch` or an `if else` construct. Having multiple `if`s here can confuse. I understand it saves a line of code though.